portal/templates/reporting_dashboard.html

```diff
@@ -6,6 +6,9 @@ <h3 class="tnth-headline">{{ _("Reporting Dashboard") }}</h3>
 
     <h2 class="tnth-headline">{{ _("Usage Statistics") }}</h2>
     <div>
+        <p>{{ _("Registrations are collected from the User.registration timestamps of all Users without the Test Role") }}</p>
```

Comment:
It looks like we're recording the time of account creation rather than registration in user.registered.  Those would be the same for an account created during registration, but incorrect for accounts created by staff.  Bug in `user.promote_to_registered`